handbook/product/README.md

```diff
@@ -250,44 +199,69 @@ We track competitors' capabilities and adjacent (or commonly integrated) product
-- Fleet always prioritizes bugs.
+Requests are weighed by:
+- The completeness of the request (see [making a request](#making-a-request)
+- How urgent the need is for the customer
+- How much impact the request will have. This may be a wide impact across many customers and/or high impact on one
+- How well the request fits within Fleet's product vision and roadmap
+- Whether the feature seems like it can be designed, estimated, and developed in 6 weeks, given its individual complexity and when combined with other work already accepted
 
-### Making a request
+### After the feature is accepted
+- After the 🎁🗣 Feature Fest meeting, the Head of Product resets the agenda to blank by doing the following:
+  - Create issues for accepted items
+  - Notify absent requesters of decisions
+  - Move that week's feature requests to the backup journal document
 
-To make a request or advocate for a request from a customer or community member,  Fleet asks all members of the organization to add their name and a description of the request to the list in the [🗣  Feature Fest agenda](https://docs.google.com/document/d/1mwu5WfdWBWwJ2C3zFDOMSUC9QCyYuKP4LssO_sIHDd0/edit#heading=h.zahrflvvks7q).
-Then attend the next scheduled 🎁🗣 Feature Fest meeting.
+> The product team's commitment to the requester is that a prioritized feature will be delivered within 6 weeks or the requester will be notified within 1 business day of the decision to de-prioritize the feature. 
 
-Anyone in the company is invited to attend the 🎁🗣 Feature Fest meeting.  Folks from the wider community can also [request an invite](https://fleetdm.com/contact). 
+Potential reasons for why a feature may be de-prioritized include:
+- The work was not designed in time. Since Fleet's engineering sprints are 3 weeks each, this means that a prioritized feature has 3 weeks to be designed, approved, and estimated in order to make it to the engineering sprint. At the prioritization meeting, the perceived design complexity of proposed features will inevitably be different from the actual complexity. 
+  - This may be because other higher-priority design work took longer than expected or the work itself was more complex than expected
+- The was designed but was not selected for the sprint. When a new sprint starts, it is populated with bugs, features, and technical tasks. Depending on the size and quantity of non-feature work, certain features may not be selected for the sprint.
 
-Requests are weighed from top to bottom while prioritizing attendee requests.  This means that if the individual that added a feature request is not in attendance, the feature request will be discussed towards the end of the call if there's time.
+Whenever a feature is de-prioritized, it only means that the feature has been _de-prioritized_ at that time. It is up to the requester to bring the request back again at another 🎁🗣 Feature Fest meeting.
```

Comment:
I think it bears repeating because this is in the context of after the feature has been intook. But I think being explicit about that here would be good.

```suggestion
Just as when a feature request is not accepted in the 🎁🗣 Feature Fest meeting, whenever a feature is de-prioritized after it has been accepted, it only means that the feature has been _de-prioritized at this time_. It is up to the requester to bring the request back again at another 🎁🗣 Feature Fest meeting.
```